amy/autoemails/models.py

```diff
@@ -289,6 +289,10 @@ class Trigger(ActiveMixin, CreatedUpdatedMixin, models.Model):
         ("workshop-request-response1", "Response to Workshop Request 1"),
         ("workshop-request-response2", "Response to Workshop Request 2"),
         ("workshop-request-response3", "Response to Workshop Request 3"),
+        (
+            "consent-required",
```

Comment:
@pbanaszkiewicz could you tell me what a trigger is used for vs an action? Looks like the action classes construct the email. What do the triggers represent?

Reply:
Trigger represents a cause for action.
Action is not a model, it's a Python object, so we needed a way to mark what users can do that would result in email sent.